src/components/project-member/dto/project-member.dto.ts

```diff
@@ -0,0 +1,22 @@
+import { Field, ObjectType } from 'type-graphql';
+import { Resource, SecuredPropertyList } from '../../../common';
+import { SecuredUser } from '../../user';
+import { Role } from '../../user/role';
+
+@ObjectType({
+  description: SecuredPropertyList.descriptionFor(
+    'a role of projectMember types'
+  ),
+})
+export abstract class SecuredRoles extends SecuredPropertyList(Role) {}
+
+@ObjectType({
+  implements: [Resource],
+})
+export class ProjectMember extends Resource {
```

Comment:
`modifiedAt` can still be on this object. It's just read-only which is why it's not on the inputs.